### PR TITLE
fix: menu icon fade out

### PR DESCRIPTION
**Pull request by christoph-heinrich**
It also used `text_opacity` instead of `menu_opacity` like the rest of the title/search bar.

But maybe everything in that bar should use `text_opacity` anyway?

### CODE REVIEW

**Comment by tomasklaen** [comment]
> But maybe everything in that bar should use text_opacity anyway?

Yes, icons are essentially text.

**Comment by christoph-heinrich** [comment]
> > But maybe everything in that bar should use text_opacity anyway?
> 
> Yes, icons are essentially text.

But the other text in the title/search bar doesn't use text_opacity, hence my confusion.

**Comment by christoph-heinrich** [comment]
I've added a commit to fix the menu_opacity and text_opacity usage.
One of those is for the highlight, not sure if we want to change that one.

**Comment by tomasklaen** [comment]
> One of those is for the highlight, not sure if we want to change that one.

Since the addition of `options.menu_opacity` it's all a bit misnamed so it's hard to follow. It needs a refactor.

`options.menu_opacity` should only affect the menu background. So, we should swap current `menu_opacity` and `text_opacity` for:

- `menu_opacity` - It should start at 1 for current menu, decay for (parent/sub)menus, and multiplied by `self.opacity`. This should be the base used by everything.
- `background_opacity = menu_opacity * options.menu_opacity` - and only used by the background, so I guess we don't need it as a value.

**Comment by tomasklaen** [comment]
Don't bother with this, I'm working on the opacity options overhaul and this is in the path, so I'll include it there.

**Comment by christoph-heinrich** [comment]
Alright I'll close it then.
